### PR TITLE
Refactor index js

### PR DESCRIPTION
**Pull request by karinamzalez**
- pull all draw functions out of index.js 
- write unit tests for every draw function
- remove renderWorld
  @neight-allen (we need help testing clearRect on line 69#world-test

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -3,57 +3,14 @@ const canvas = document.getElementById('game');
...
-
-function renderWorld(){
-  drawCanvas();
-  drawTray();
-  drawOverlord();
-  moveOverlord();
-  // drawhealthFood();
-  // movehealthFood();
-}
+var world = new World(canvas.width, canvas.height, context);
```
Comment (chompasina) [inline lib/index.js:6]: Looks great without all those functions!

File: test/world-test.js [hunk 2]
```diff
@@ -58,3 +63,33 @@ describe('World', function(){
     });
   });
 });
+
+describe('draw canvas', function(){
+  var context = stub().of("fillRect");
+  // var context = stub().of("clearRect");
```
Comment (chompasina) [inline test/world-test.js:69]: Let's figure out why this doesn't work.

**Comment by chompasina** [comment]
Except for the failing test with the clearRect issue, it looks good to me. I think we should merge for now but check in with @neight-allen tomorrow.

**Comment by deborahleehamel** [comment]
Nice implementation with the stubs. I think we can merge this to keep moving forward with our new HeathFood objects feature and refactor as we go.

**Pull request merged**